certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json

```diff
@@ -0,0 +1,887 @@
+            "enum": ["ui"]
+          },
+          {
+            "description": "Zip file containing files to translate.",
+            "x-ms-summary": "Source Files",
+            "format": "binary",
+            "in": "formData",
+            "name": "sourceFile",
+            "type": "file"
+          },
+          {
+            "in": "formData",
+            "name": "trainMTWithTM",
+            "x-ms-summary": "Custom MT Model",
+            "description": "Create a custom MT system based on uploaded translation memory files.",
+            "type": "boolean",
+            "required": false,
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "description": "Zip file containing translation memory files.",
+            "x-ms-summary": "Translation Memory files",
+            "format": "binary",
+            "in": "formData",
+            "name": "tmFile",
+            "type": "file",
+            "required": false,
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "String describing the status of the request."
+                },
+                "message": {
+                  "type": "string",
+                  "description": "String elaborating on the status."
+                },
+                "translationId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "The translation request ID, used retrieve and download operations."
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "oauth2_auth": [
+              "user_impersonation"
+            ]
+          }
+        ],
+        "tags": [
+          "default"
+        ],
+        "summary": "Translate",
+        "operationId": "Translate",
+        "description": "Submit a new translation request."
+      }
+    },
+    "/dts/align/download": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "File Output",
+            "schema": {
+              "type": "file",
+              "format": "binary",
+              "x-ms-summary": "Aligned File.",
+              "description": "Binary file content from alignment output.",
+              "title": "File Output",
+              "x-ms-visibility": "important"
+            }
+          }
+        },
+        "summary": "Download Alignment Result",
+        "operationId": "Align_Download",
+        "parameters": [
+          {
+            "name": "filename",
+            "x-ms-summary": "File name",
+            "description": "Filename returned in alignment output.",
+            "x-example": "0_7e47d31c-f3e3-43ea-b6a8-986cc58861cd_1_BI.fr.label_Aligned.xlf",
```

Comment:
@joshftb secret-look-a-like-values are not allowed. Please remove them or replace with other string value.

Reply:
Done!